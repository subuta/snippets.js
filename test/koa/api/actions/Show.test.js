/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import ShowAction from 'lib/koa/api/actions/Show'

describe('koa/api/actions/Show', () => {
  it('should Show koa action', () => {
    expect(format(ShowAction({
      model: 'user',
      routeConfig: {}
    }))).toMatchSnapshot()
  })
})
