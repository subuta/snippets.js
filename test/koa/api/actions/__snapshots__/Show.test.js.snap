// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`koa/api/actions/Show should Show koa action 1`] = `
"user.get('/:id', async (ctx) => {
  const {User} = ctx.state.models
  let params = {}

  /* mat Before show [start] */
  /* mat Before show [end] */

  ctx.body = await User.query()
    .eager('')
    .findFirst({...params, id: ctx.params.id})
})
"
`;
